.github/workflows/integration-tests-release.yaml

```diff
@@ -25,13 +25,30 @@ on:
       kubernetes_version:
         description: 'Kubernetes version'     
         required: true
-        default: 'v1.20.2'
+        default: 'v1.21.1'
```

Comment:
The version in `tests/integration/scripts/create_kind_cluster.sh` is set to 1.22.2. I think we should either make the default in the workflow 1.22.2 or change the version in the script to 1.21.1.

Reply:
I'll change the version in the script then, because changing the default in the workflow would mean not executing the upgrade tests on PRs.